app/config/FrontendAppConfig.scala

```diff
@@ -42,6 +42,9 @@ class FrontendAppConfig @Inject() (servicesConfig: ServicesConfig, configuration
   val sdilBaseUrl: String = servicesConfig.baseUrl("soft-drinks-industry-levy")
   val sdilHomeUrl: String = servicesConfig.getString("sdilHomeUrl")
 
+  val sdilAccountFrontendBaseUrl: String = servicesConfig.baseUrl("soft-drinks-industry-levy-account-frontend")
+  val accountFrontendHomeUrl: String = s"$sdilAccountFrontendBaseUrl/soft-drinks-industry-levy-account-frontend/home"
+
```

Comment:
We shouldn't need this as we have the sdilHomeUrl. For some period of time returns is going to be LIVE when account-frontend isn't so any redirects to home will be the old service hence why we have the sdilHomeUrl